org.bridgedb.uri.api/pom.xml

```diff
@@ -48,7 +48,43 @@
                 <version>${hamcrest.version}</version>
                 <scope>test</scope>
             </dependency>
-	</dependencies>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
+            <version>RELEASE</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
+            <version>RELEASE</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
+            <version>RELEASE</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
+            <version>RELEASE</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
+            <version>5.5.0-RC1</version>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
```

Comment:
I missed that, I have fixed it now.

Reply:
No worries. Make it a routine to always look at the commits you made: peer review them yourself before you submit them. It's all about routine.

Reply:
PS, that's why it was important to me you to start showing results instead of just talking. With talking we have years and years of routine, with open source code development we are all learning :)